dev/io.openliberty.netty.internal.impl/src/io/openliberty/netty/internal/tcp/TCPConfigurationImpl.java

```diff
@@ -0,0 +1,1285 @@
+                    keyType = ValidateUtils.KEY_TYPE_BOOLEAN;
+                    oldBool = this.allocateBuffersDirect;
+                    if (convertBooleanValue(value) != oldBool) {
+                        result = ValidateUtils.VALIDATE_NOT_EQUAL;
+                    }
+                    continue;
+                }
+
+                if (key.equalsIgnoreCase(KEEP_ALIVE)) {
+                    // convert and check
+                    keyType = ValidateUtils.KEY_TYPE_BOOLEAN;
+                    oldBool = this.keepAlive;
+                    if (convertBooleanValue(value) != oldBool) {
+                        result = ValidateUtils.VALIDATE_NOT_EQUAL;
+                    }
+                    continue;
+                }
+
+                if (key.equalsIgnoreCase(RCV_BUFF_SIZE)) {
+                    // convert and check
+                    keyType = ValidateUtils.KEY_TYPE_INT;
+                    oldValue = this.receiveBufferSize;
+                    if (convertIntegerValue(value) != oldValue) {
+                        result = ValidateUtils.VALIDATE_NOT_EQUAL;
+                    }
+                    continue;
+                }
+
+                if (key.equalsIgnoreCase(SEND_BUFF_SIZE)) {
+                    // convert and check
+                    keyType = ValidateUtils.KEY_TYPE_INT;
+                    oldValue = this.sendBufferSize;
+                    if (convertIntegerValue(value) != oldValue) {
+                        result = ValidateUtils.VALIDATE_NOT_EQUAL;
+                    }
+                    continue;
+                }
+
+                if (key.equalsIgnoreCase(GROUPNAME)) {
+                    keyType = ValidateUtils.KEY_TYPE_STRING;
+                    strOldValue = this.workGroupName;
+                    this.workGroupName = (String) value;
+                    if (null == this.workGroupName) {
+                        result = ValidateUtils.VALIDATE_ERROR;
+                    }
+                    continue;
+                }
+
+                if (key.equalsIgnoreCase(INACTIVITY_TIMEOUT)) {
+                    // convert and check
+                    keyType = ValidateUtils.KEY_TYPE_INT;
+                    inactivityTimeoutNew = convertIntegerValue(value);
+                    result = ValidateUtils.testInactivityTimeout(inactivityTimeoutNew);
+
+                    continue;
+                }
+
+                if (key.equalsIgnoreCase(COMM_OPTION)) {
+                    // convert and check
+                    keyType = ValidateUtils.KEY_TYPE_INT;
+                    oldValue = this.commOption;
+                    if (convertIntegerValue(value) != oldValue) {
+                        result = ValidateUtils.VALIDATE_NOT_EQUAL;
+                    }
+                    continue;
+                }
+
+                if (key.equalsIgnoreCase("chainData")) {
+                    // this parameter is made available by the channel framework
```

Comment:
'channel framework' seems odd and so does 'this channel' - is this really any channel, TCP channels?

Reply:
I wanted to preserve config parsing as closely as possible for this PR